ome_zarr/reader.py

```diff
@@ -354,7 +354,9 @@ def __init__(self, node: Node) -> None:
             node.metadata["visible"] = visibles
             node.metadata["contrast_limits"] = contrast_limits
             node.metadata["colormap"] = colormaps
+
         except Exception as e:
+            raise e
```

Comment:
Should this be there? The LOGGER line below will never be called

Reply:
Pretty likely debugging code that crept in. Thanks, @glyg